pkg/zrouter/zmiddlewares/logging.go

```diff
@@ -20,8 +20,9 @@ func LoggingMiddleware(next http.Handler) http.Handler {
 		next.ServeHTTP(rw, r)
 		duration := time.Since(start)
 		requestID := r.Header.Get(RequestIDHeader)
+		ctx := r.Context()
 
-		zap.S().Debugf("request_id: %s - Method: %s - URL: %s | Status: %d - Duration: %s - Response Body: %s",
+		logger.Log().Debugf(ctx, "request_id: %s - Method: %s - URL: %s | Status: %d - Duration: %s - Response Body: %s",
```

Comment:
If we do so, there is no need for the request_id, as the logger will print it inside the fields...